### ISSUE TITLE
Remove fake dependency from libdrm

### ISSUE THREAD

**Issue by dvrogozh** [intel/gmmlib#8]
At this place and around it:
https://github.com/intel/gmmlib/blob/2eea1a175ab4f3388925c91c870214f36f17f20d/Source/GmmLib/CMakeLists.txt#L183

gmmlib introduces fake dependency from _custom_ version of libdrm. This is error prone since users may have patched version of libdrm (from MSS package) or have non-patched version. As a result there is a very high chance of building invalid version of gmmlib.

Could you, please, do the following:
1. Completely remove any dependency from libdrm in gmmlib
2. Move intel_mocs_table.h inside gmmlib
3. Introduce configure level option to request build with extended mocs table instead of open source one
4. By default build against open source mocs table

In other words, I suggest to do something similar to the following:
```
# cmake -DUSE_EXTENDED_MOCS_TABLE=TRUE ..
# cat ./Source/GmmLib/CMakeLists.txt
…
if (NOT DEFINED USE_EXTENDED_MOCS_TABLE)
  set(USE_EXTENDED_MOCS_TABLE FALSE)
endif()

if (USE_EXTENDED_MOCS_TABLE)
  add_definitions(-DUSE_EXTENDED_MOCS_TABLE)
  # no need in additional include dir since intel_mocs_table.h is inside gmmlib tree
endif()
…
```

**Comment by drprajap** [comment]
This will be addressed by next week. Plan is to address all 4 points you mentioned here.

**Comment by drprajap** [comment]
This issue has been addressed by [a2fd856](https://github.com/intel/gmmlib/commit/a2fd8568eaabec5ee8368746a17594bf95318648)
